InterfaceTPL/tsgBlasWrappers.hpp

```diff
@@ -196,7 +201,75 @@ namespace TasBLAS{
                 throw std::runtime_error(std::string("Lapack zgels_ infer-worksize-stage exited with code: ") + std::to_string(info));
         }
     }
-    #ifdef Tasmanian_BLAS_HAS_ZGELQ
+    /*!
+     * \ingroup TasmanianTPLWrappers
+     * \brief LAPACK dstebz
+     */
+    inline void stebz(char range, char order, int N, double vl, double vu, int il, int iu, double abstol, double D[], double E[],
+                      int M, int nsplit, double W[], int iblock[], int isplit[], double work[], int iwork[]) {
```

Comment:
There is an inconsistency here. `M` is an output of the eigensolver, but you are passing in by value so the C++ wrapper will not return the value.

Reply:
Changed. I also made the same correction for ``nsplit``.